### DISCUSSION TITLE
Unpublish an already published eForm

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#536] [Q&A] [answered]
Is there a way to unpublish an already published eForm?

With the old system, an F03 withdrawal was sent to unpublish them.

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
Any notice on TED stays published in the OJS "forever" (online for 10 years), under current forms and eForms.

In eForms you can change any notice that has been published. In extreme cases, the change can "cancel" the published notice but this is not recommended as it may have consequences for the procedure.

If a notice hasn't been published yet (still in Submitted status), you can request to stop its publication.

This isn't a technical question or about the SDK - please see the presentation on notice lifecycle of 1 February for more details - https://op.europa.eu/en/web/ted-eforms/agenda

**Reply by AlexAS81** [reply]
And how can I "cancel" the published notice? What should be indicated in the eForm?

**Comment by ferrakaop** [comment]
To "cancel" a notice, you send a Change notice with Cancel as the reason code.  The notice is published again in TED (like all Change notices) but it renders the notice logically and legally null and void.  

This is why we don't recommend using Cancel except in exceptional cases and only if you're very sure about what you are doing.  Note that cancelling a notice does not cancel the procedure or lot - this is done via a contract award notice with no winner.  again, please consult the February presentation for a fuller overview.

This question doesn't belong in this technical forum.
